etc/c/odbc/sql.d

```diff
@@ -2,8 +2,8 @@
 Declarations for interfacing with the ODBC library.
 
 Adapted with minimal changes from the work of David L. Davis
-(refer to the $(WEB
-forum.dlang.org/post/cfk7ql$(DOLLAR)1p4n$(DOLLAR)1@digitaldaemon.com,
+(refer to the $(LINK2
+http://forum.dlang.org/post/cfk7ql$(DOLLAR)1p4n$(DOLLAR)1@digitaldaemon.com,
```

Comment:
Unfortunately, this is no good. Ddoc interprets "http:" as a section. Can't have a break between "LINK2" and "http:".